models/calendar.yml

```diff
@@ -83,7 +83,17 @@
 
     - 7:45 PM: Speaker TBA
 
-    - 8:00 PM: Speaker TBA
+    - 8:00 PM:
+        type: Talk
+        title: Asynchronous functions
+        person:
```

Comment:
Done, but I had to go look up my Twitter ID because I haven't used it in years.
Why is this required?